src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -292,17 +298,30 @@ void ResourceAllocationManagerImpl::SetResourceAquired(
     const std::string& module_type, const uint32_t app_id) {
   LOG4CXX_AUTO_TRACE(logger_);
   allocated_resources_[module_type] = app_id;
-  SendOnRCStatusNotification();
+  SendOnRCStatusNotification(false);
 }
 
-void ResourceAllocationManagerImpl::SendOnRCStatusNotification() {
+void ResourceAllocationManagerImpl::SendOnRCStatusNotificationToMobile() {
   LOG4CXX_AUTO_TRACE(logger_);
   auto rc_apps = RCRPCPlugin::GetRCApplications(app_mngr_);
   for (const auto& rc_app : rc_apps) {
-    auto msg_to_mobile = CreateOnRCStatusNotificationToMobile(rc_app);
+    auto msg_to_mobile = CreateOnRCStatusNotificationToMobile(rc_app, true);
     rpc_service_.SendMessageToMobile(msg_to_mobile);
-    auto msg_to_hmi = CreateOnRCStatusNotificationToHmi(rc_app);
-    rpc_service_.SendMessageToHMI(msg_to_hmi);
+  }
+}
+
+void ResourceAllocationManagerImpl::SendOnRCStatusNotification(
+    const bool include_allowed) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  auto rc_apps = RCRPCPlugin::GetRCApplications(app_mngr_);
+  for (const auto& rc_app : rc_apps) {
+    auto msg_to_mobile =
+        CreateOnRCStatusNotificationToMobile(rc_app, include_allowed);
```

Comment:
@LitvinenkoIra you have function SendOnRCStatusNotificationToMobile, but you does not use it here.
You use creating and manual sending, that is confusing.

Reply:
@LuxoftAKutsan I use SendOnRCStatusNotificationToMobile in case when I need to send notification only to mobile and with allowed param

Reply:
@LitvinenkoIra it is not obvious and easy to make mistake during changing this code